frontend/sw360-portlet/src/main/resources/META-INF/resources/html/projects/includes/attachmentSelectTable.jspf

```diff
@@ -216,6 +227,201 @@
     </core_rt:if>
     </tbody>
 </table>
+</div>
+
+<div class="tab-pane fade" id="pills-onlyapprovedView" role="tabpanel" aria-labelledby="pills-onlyapproved-tab">
+<table class="table" id="LinkedProjectsInfoForOnlyApprovedAttachment" style="width: 100%; max-width: 100%;"
```

Comment:
This table is mostly duplicate of the other one.

Reply:
Thanks @smrutis1 for the review, I tried to reused the existing table with some filter condition but it is breaking the only approved filter functionality, also since this jspf is been used by the  **GenerateSourceCodeBundle**  view as well, i am afraid it might impact that page, so i created the new table for only approved and made all the function call separately via this new table id.
Please let me know your concern.

Reply:
Thanks for explaining challenges.